stablehlo/reference/Element.cpp

```diff
@@ -607,6 +601,203 @@ Element ceil(const Element &el) {
   return Element(el.getType(), val);
 }
 
+Element convert(Type type, bool value) {
+  if (isSupportedBooleanType(type)) return Element(type, value);
+  if (isSupportedIntegerType(type))
+    return Element(type, APInt(type.getIntOrFloatBitWidth(), value ? 1 : 0,
+                               type.cast<IntegerType>().isSigned()));
```

Comment:
We don't need the isSigned parameter here because the val parameter is guaranteed to be non-negative.